src/helpers/programHelper.js

```diff
@@ -1184,12 +1186,60 @@ class ProgramServiceHelper {
             return reject(err.message);
           })
         }
-        else {                    
+        else {
           return resolve(cacheData);
         }
       })
     })
   }
+
+
+  /**
+   * Is user already nominated
+   * @param {*} program_id  program id
+   * @param {*} orgosid     Open saber org id
+   * @returns boolean
+   */
+  async isAlreadyNominated(program_id, orgosid) {
```

Comment:
@ahghatol  can you make this generalised for org as well as indidual?

Reply:
Done